test/test_config.rb

```diff
@@ -317,6 +317,10 @@ def test_run_hooks_before_fork
     assert_run_hooks :before_fork
   end
 
+  def test_run_hooks_on_booted
+    assert_run_hooks :on_booted
```

Comment:
This seems to be failing inside of `assert_run_hooks`. I think it is specifically this line:
https://github.com/puma/puma/blob/a6cdca85a15b2c277f64a5fa7572effb786fa9a1/test/test_config.rb#L379

Reply:
This is leftover from the initial implementation